test/api/SettingTest.ts

```diff
@@ -21,6 +22,98 @@ chai.use(responseHelper);
 const testData: TestData = new TestData();
 let initialTenant: Tenant;
 
+// Update pricing setting to have sensitive data to test on it
+async function updatePricingWithSensitiveData():Promise<any> {
```

Comment:
It's more than just an update operation, it also includes outputs validation for mocha. Its name shall reflect it.